transforms/universal/hap/python/Makefile

```
@@ -0,0 +1,62 @@
+# Define the root of the local git clone for the common rules to be able 
+# know where they are running from.
+REPOROOT=../../../..
+# Include a library of common .transform.* targets which most
+# transforms should be able to reuse.  However, feel free
+# to override/redefine the rules below. 
+include $(REPOROOT)/transforms/.make.transforms
+
+#TRANSFORM_RUNTIME_SRC_FILE=hap_transform_python.py
+TRANSFORM_NAME=hap
+
+include $(REPOROOT)/transforms/.make.transforms
```

Line 12 is repeating what is on line 7. It should be removed.
```suggestion
HAP_PYTHON_VERSION= $(DPK_VERSION)
```